eval/fixtures/golden_cases/case-006/bundle/files/config.py

```
@@ -0,0 +1,18 @@
+
+
+def get_headers():
+    return {
+        "Authorization": f"Bearer {API_KEY}",
+        "Content-Type": "application/json",
+    }
+
+
+def fetch_data(endpoint):
+    url = f"{API_BASE}/{endpoint}"
+    resp = requests.get(url, headers=get_headers())
```

Ruff (0.15.10)

[error] Probable use of `requests` call without timeout

(S113)